app/views/migration/_form.html.erb

```diff
@@ -20,15 +20,13 @@
           <%= cho.simple_fields_for(:dcterms_spatial_places) do |place| %>
             <%= place.input(:owl_sameAs, as: :autocomplete, url: vocabularies_europeana_places_path, param: 'q', label: i18n.t('labels.edm_aggregatedCHO.dcterms_spatial_places.owl_sameAs.' + (place.object == cho.object.dcterms_spatial_places.first ? 'begin' : 'end')), input_html: { 'data-deref-url': vocabularies_europeana_places_dereference_path } ) %>
           <% end %>
+
+          <h2 class="legend"><%= i18n.t('legends.subject_agent') %></h2>
```

Comment:
this h2 is right aligned, but if the above fieldset is closed and this section is made it's own fieldset with a legend instead of a heading it looks fine.

```
-        <h2 class="legend"><%= i18n.t('legends.subject_agent') %></h2>
+        <% end %>
+        <%= field_set_tag(i18n.t('legends.subject_agent')) do %>
```